crates/bevy_solari/src/scene/mod.rs

```diff
@@ -0,0 +1,90 @@
+mod binder;
+mod blas;
+mod extract;
+mod types;
+
+pub use binder::RaytracingSceneBindings;
+pub use types::RaytracingMesh3d;
+
+use crate::SolariPlugin;
+use bevy_app::{App, Plugin};
+use bevy_asset::{load_internal_asset, weak_handle, Handle};
+use bevy_ecs::schedule::IntoScheduleConfigs;
+use bevy_render::{
+    extract_resource::ExtractResourcePlugin,
+    mesh::{
+        allocator::{allocate_and_free_meshes, MeshAllocator},
+        RenderMesh,
+    },
+    render_asset::prepare_assets,
+    render_resource::{BufferUsages, Shader},
+    renderer::RenderDevice,
+    ExtractSchedule, Render, RenderApp, RenderSystems,
+};
+use binder::prepare_raytracing_scene_bindings;
+use blas::{prepare_raytracing_blas, BlasManager};
+use extract::{extract_raytracing_scene, StandardMaterialAssets};
+use tracing::warn;
+
+const RAYTRACING_SCENE_BINDINGS_SHADER_HANDLE: Handle<Shader> =
+    weak_handle!("9a69dbce-d718-4d10-841b-4025b28b525c");
+const SAMPLING_SHADER_HANDLE: Handle<Shader> = weak_handle!("a3dd800b-9eaa-4e2b-8576-80b14b521ae9");
+
+pub struct RaytracingScenePlugin;
```

Comment:
Perhaps because English is not my native language, but personally, I think `RayTracingScenePlugin` is a better naming format than `RaytracingScenePlugin`. For example, the abbreviation for "Ray Traced Global Illumination" is "RTGI" instead of "RGI." Writing it as "RayTracing" feels more natural and comfortable to me.